code/game/objects/items/cards_ids.dm

```diff
@@ -435,17 +435,20 @@ update_label("John Doe", "Clowny")
 				to_chat(usr, "<span class='notice'>The card's microscanners activate as you pass it over the ID, copying its access.</span>")
 
 /obj/item/card/id/syndicate/attack_self(mob/user)
-	var/static/valid_jobs = list( // copy-paste from hop card paint machine, but this has more options, and valid to be static.
-			"----Command----","Command (Custom)", "Captain", "Acting Captain",
-			"----Service----","Service (Custom)", "Assistant", "Head of Personnel", "Bartender", "Cook", "Botanist", "Janitor", "Curator",
-			"Chaplain", "Lawyer", "Clown", "Mime", "Barber", "Stage Magician",
-			"----Cargo----","Cargo (Custom)","Quartermaster", "Cargo Technician","Shaft Miner",
-			"----Engineering----","Engineering (Custom)","Chief Engineer", "Station Engineer", "Atmospheric Technician",
-			"----Science----","Science (Custom)","Research Director", "Scientist", "Roboticist", "Exploration Crew",
-			"----Medical----","Medical (Custom)","Chief Medical Officer", "Medical Doctor", "Chemist", "Geneticist", "Virologist", "Paramedic", "Psychiatrist",
-			"----Security----","Security (Custom)","Head of Security", "Warden", "Detective", "Security Officer", "Brig Physician", "Deputy",
-			"----MISC----","Unassigned","Prisoner","CentCom (Custom)","CentCom","ERT","VIP","King","Syndicate","Clown Operative"
-		)
+	var/valid_jobs = list( // copy-paste from hop card paint machine, but this has more options, and valid to be static.
+		"----Command----", "Command (Custom)",JOB_NAME_CAPTAIN,
+		"----Service----", "Service (Custom)", JOB_NAME_ASSISTANT, JOB_NAME_HEADOFPERSONNEL, JOB_NAME_BARTENDER, JOB_NAME_COOK,
+			JOB_NAME_BOTANIST, JOB_NAME_JANITOR, JOB_NAME_CURATOR,JOB_NAME_CHAPLAIN, JOB_NAME_LAWYER,
+			JOB_NAME_CLOWN, JOB_NAME_MIME, JOB_NAME_BARBER, JOB_NAME_STAGEMAGICIAN,
+		"----Cargo----","Cargo (Custom)",JOB_NAME_QUARTERMASTER, JOB_NAME_CARGOTECHNICIAN,JOB_NAME_SHAFTMINER,
+		"----Engineering----","Engineering (Custom)",JOB_NAME_CHIEFENGINEER, JOB_NAME_STATIONENGINEER, JOB_NAME_ATMOSPHERICTECHNICIAN,
+		"----Science----","Science (Custom)",JOB_NAME_RESEARCHDIRECTOR, JOB_NAME_SCIENTIST, JOB_NAME_ROBOTICIST, JOB_NAME_EXPLORATIONCREW,
+		"----Medical----","Medical (Custom)",JOB_NAME_CHIEFMEDICALOFFICER, JOB_NAME_MEDICALDOCTOR, JOB_NAME_CHEMIST, JOB_NAME_GENETICIST,
+			JOB_NAME_VIROLOGIST, JOB_NAME_PARAMEDIC, JOB_NAME_PSYCHIATRIST,
+		"----Security----","Security (Custom)",JOB_NAME_HEADOFSECURITY, JOB_NAME_WARDEN, JOB_NAME_DETECTIVE, JOB_NAME_SECURITYOFFICER,
+			JOB_NAME_BRIGPHYSICIAN, JOB_NAME_DEPUTY,
+		"----MISC----","Unassigned","Prisoner","CentCom (Custom)","CentCom","ERT",JOB_NAME_VIP,"King","Syndicate","Clown Operative"
```

Comment:
Prisoner and King have 1:1 defines, as do CentCom and ERTs (might require changes elsewhere to use those though, they're not 1:1)

Also this should tie more directly in to the `JOB_HUD` defines somehow, but for now it's fine

Reply:
Those are not 'real' jobs, so it's made as job defines. JOB_HUD isn't made for job name ones because real jobs do have capital at their first letter while JOB_HUD ones aren't.

Reply:
There are JOB_NAME defines for Prisoner and King though. That's what I'm saying.

I searched the code, there is literally `JOB_NAME_PRISONER`

Reply:
Also I am saying more that this list should be directly tied in some sort of list relating JOB_NAME to JOB_HUD instead of whatever amalgam this currently is - but for now it's fine.

Reply:
ah, you're right. I thought I didn't add those. also, JOB_NAME is tied to JOB_HUD already. Thereis a proc `get_hud_by_jobname(job_name)` that returns a hud style based on job name